src/context/ColonyContext.tsx

```diff
@@ -14,12 +14,16 @@ interface ColonyContextValue {
   loading: boolean;
   canInteractWithColony: boolean;
   refetchColony: (() => null) | ObservableQuery['refetch'];
+  startPolling: (pollInterval: number) => void;
+  stopPolling: () => void;
 }
 
 const ColonyContext = createContext<ColonyContextValue>({
   loading: false,
   canInteractWithColony: false,
   refetchColony: () => null,
+  startPolling: () => undefined,
+  stopPolling: () => undefined,
```

Comment:
Exposing the polling functions allows us to poll after clicking Claim All, which is the only way to ensure the Stakes Tab updates automatically without needing a browser refresh.